src/quadratic_diff.jl

```diff
@@ -281,7 +281,7 @@ function reverse_differentiate!(model::QPDiff)
     dλ = partial_grads[nv+1:nv+nineq]
     dν = partial_grads[nv+nineq+1:end]
 
-    model.back_grad_cache = QPForwBackCache(dz, dλ, dν)
+    model.back_grad_cache = QPForwRevCache(dz, dλ, dν)
```

Comment:
QP here is a bit inconsistent with naming elsewhere.

Reply:
changed